src/test/compile-fail/macro-at-most-once-rep-ambig.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
```

Comment:
I always appreciate a comment here explaining what this test is trying to do. Example:

---

Test that in `$(a)?+`, the `?` is interpreted as a separator, but not for `$(a)?`. This test focuses on error cases.